### PR TITLE
DDF-UI-212 Move SecureWebSocketServlet into DDF

### PR DESCRIPTION
**Pull request by bakejeyner**
[DDF Pull Request](https://github.com/codice/ddf/pull/6068).

### CODE REVIEW

**Comment by bakejeyner** [comment]
CI should be delayed until the DDF PR is merged.

File: pom.xml [hunk 1]
```diff
@@ -51,7 +51,7 @@
     <maven.deploy.plugin.version>2.8.2</maven.deploy.plugin.version>
 
     <!--Backend properties-->
-    <ddf.version>2.24.0</ddf.version>
+    <ddf.version>2.25.0-SNAPSHOT</ddf.version>
```
Comment (blen-desta) [inline pom.xml]: You should change this to `2.25.0` instead

**Review by SmithJosh — APPROVED** [review]

**Review by blen-desta — APPROVED** [review]
Hero'd successfully. See: https://github.com/codice/ddf/pull/6068#issuecomment-632918383

**Review by parret — APPROVED** [review]

**Comment by bakejeyner** [comment]
build now

**Comment by bakejeyner** [comment]
CI should be delayed until the next release of DDF

**Review by bdeining — APPROVED** [review]

**Comment by jlcsmith** [comment]
@bakejeyner - DDF has been released so this should be unblocked.  Please update the dependency to DDF 2.25.1.

**Comment by bakejeyner** [comment]
build now

**Comment by vinamartin** [comment]
Merged in https://github.com/codice/ddf-ui/pull/282

**Pull request closed without merging**